roles/os_zabbix/library/zbxapi.py

```diff
@@ -17,11 +21,22 @@
 #  Purpose: An ansible module to communicate with zabbix.
 #
 
+# pylint: disable=line-too-long
+# Disabling line length for readability
```

Comment:
You're not allowed to disable pylint checks for the whole file, only on individual lines.

See the best practices guide:
https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#pylint